daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTest.scala

```diff
@@ -24,6 +24,24 @@ class SpeedyTest extends WordSpec with Matchers {
   import SpeedyTest._
   import defaultParserParameters.{defaultPackageId => pkgId}
 
+  "free variables" should {
+
+    val pkgs = typeAndCompile(p"")
+
+    "be captured correctly" in {
```

Comment:
Given how simple it is to write such a test I'd be happy to see a test like
```
(\ (a: Int64) (b: Int64) -> SUB_INT64 a b) 88 33
```
as well to make sure we also get the order of the arguments right. I'm fine with doing this in a separate PR.